app/views/hub/clients/show.html.erb

```diff
@@ -251,30 +226,56 @@
             <h2 class="text--bold"><%= t(".other_info") %></h2>
+            <div class="field-display">
+              <span class="form-question">Intake type:</span>
+              <span class="label-value">
+                <%= @client.intake.drop_off? ? "Drop off" : "Online" %>
+              </span>
+            </div>
             <div class="field-display">
               <span class="form-question"><%= t(".routing_method") %>:</span>
               <span class="label-value">
-              <%= readable_routing_method(@client) || t("general.NA") %>
-            </span>
+                <%= readable_routing_method(@client) || t("general.NA") %>
+              </span>
             </div>
             <div class="field-display">
               <span class="form-question"><%= t(".source_param") %>:</span>
               <span class="label-value">
-              <%= @client.intake&.source || t("general.NA") %>
-            </span>
+                <%= @client.intake&.source || t("general.NA") %>
+              </span>
+            </div>
+            <div class="field-display">
+              <span class="form-question">Current Intake Step:</span>
+              <span class="label-value">
+                <%= @client.intake.current_step || t("general.NA") %>
+              </span>
             </div>
           </div>
 
           <!-- Identity verification -->
-          <% if @client.intake.is_ctc? %>
-            <div class="client-profile__field-group">
-              <h2 class="text--bold">Navigator Name</h2>
+          <div class="client-profile__field-group">
+            <h2 class="text--bold">Navigator Info</h2>
+            <div class="field-display">
+              <span class="form-question">Navigator type:</span>
+              <span class="label-value"><%= @client.intake.navigator_display_names.present? ? @client.intake.navigator_display_names : t(".navigator_types_empty") %></span>
+            </div>
+            <% if @client.intake.is_ctc? %>
               <div class="field-display">
-                <span class="form-question">Legal Name:</span>
-                <span class="label-value"><%= @client.intake.navigator_name %> </span>
+                <span class="form-question">Navigator Name:</span>
+                <span class="label-value"><%= @client.intake.navigator_name || "N/A" %> </span>
               </div>
-            </div>
-          <% end %>
+            <% end %>
+            <% if @client.intake.drop_off? %>
+              <div class="field-display">
+                <span class="form-question">Photo ID Used:</span>
+                <span class="label-value"><%= @client.intake.photo_id_display_names || "N/A"%></span>
+              </div>
+              <div class="field-display">
+                <span class="form-question">Taxpayer ID Used:</span>
+                <span class="label-value"><%= @client.intake.taxpayer_id_display_names || "N/A" %></span>
+              </div>
+            <% end %>
```

Comment:
seems like these fields are specific to drop off _and_ ctc

Reply:
I originally had the same feeling, but it actually breaks if it's not scoped down to only ctc dropoff clients. These are not fields that get recorded for regular drop off clients (only for CTC ones), and when they arent scoped to just ctc dropoff clients they break because the methods are written onto Intake::CtcIntake.

Reply:
oh duh. Yes! You are correct -- I'd originally had it scoped incorrectly (which you rightly pointed out) which caused the text failures, and only responded after I'd updated thinking you were suggesting the opposite.